server/controllers/itemController.js

```diff
@@ -141,6 +143,32 @@ async function getTurnings (req, res) {
     }
 }
 
+async function getBracelets (req, red) {
```

Comment:
I think this is a typo?
```suggestion
async function getBracelets (req, res) {
```